Resources/doc/internals/configuring_the_http_client.md

```diff
@@ -27,4 +27,93 @@ hwi_oauth:
 
 If you don't want to use the HTTPlug Bundle, the client and factory services **must be specified**.
 
+## Difference since 0.6+ version
+
+On v0.5 and below, you was able to configure your http client like that:
+
+```yaml
+# app/config/config.yml
+
+hwi_oauth:
+    http_client:
+        timeout:       10 # Time in seconds, after library will shutdown request, by default: 5
+        verify_peer:   false # Setting allowing you to turn off SSL verification, by default: true
+        ignore_errors: false # Setting allowing you to easier debug request errors, by default: true
+        max_redirects: 1 # Number of HTTP redirection request after which library will shutdown request,
```

Comment:
- "requests"
- what does "shutdown" request means exactly?

Reply:
This is copy/paste form old doc. I won't touch anything here.